lib/cli-engine.js

```diff
@@ -273,6 +273,17 @@ function CLIEngine(options) {
      */
     this.options = assign(Object.create(defaultOptions), options || {});
 
+    /**
+	 * fileCache used to not operate on files that haven't changed
+     * since last successful execution
+     * @type {Object}
+     */
+    this.fCache = require("file-entry-cache").create(path.resolve(".eslintcache"));
```

Comment:
So you're saying not creating the cache file all the time? Since we need to destroy the cache if the call was not done using `--cache`, it was easier to delegate the deletion of the file to the same module instead of including `rimraf` as a dependency. if that is not a problem, them I can create the cache only if the `--cache` flag is set